pkg/webhook/conversion/testData/pkg/apis/addtoscheme_jobs_v1.go

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
move this under `example/` (or just use `example/`), just to be certain we don't cause import issues

Reply:
Done.